xray/utils/resultwriter.go

```diff
@@ -55,19 +61,19 @@ func PrintScanResults(results []services.ScanResponse, errors []formats.SimpleJs
 			return err
 		}
 		if includeLicenses {
-			err = PrintLicensesTable(licenses, isMultipleRoots, printExtended)
+			err = PrintLicensesTable(licenses, printExtended)
 		}
 		return err
 	case SimpleJson:
-		jsonTable, err := convertScanToSimpleJson(results, errors, includeVulnerabilities, isMultipleRoots, includeLicenses)
+		jsonTable, err := convertScanToSimpleJson(results, errors, includeVulnerabilities, isMultipleRoots, includeLicenses, false)
 		if err != nil {
 			return err
 		}
 		return printJson(jsonTable)
 	case Json:
 		return printJson(results)
 	case Sarif:
-		sarifFile, err := GenerateSarifFileFromScan(results, includeVulnerabilities, isMultipleRoots)
+		sarifFile, err := GenerateSarifFileFromScan(results, includeVulnerabilities, isMultipleRoots, false)
```

Comment:
It looks like the `GenerateSarifFileFromScan` can be made private.

Reply:
Frogbot uses this function